src/substrate-bridge.md

```diff
@@ -0,0 +1,224 @@
+    S ->> S: Burn (sidechain assets) / Reserve (thischain assets)¹
+    S ->> S: Add message to commitment
+    S ->> R: Prepare commitment
+    R ->> R: Sign commitment
+    R ->> S: Approve commitment
+    S ->> R: Get signatures
+    R ->> P: Submit commitment
+    P ->> A: Send tokens via XCM
+```
+
+_¹ A sidechain asset (registered through [register_sidechain_asset](https://sora-xor.github.io/sora2-network/master/doc/substrate_gen/runtime/runtime_types/parachain_bridge_app/pallet/enum.Call.html#variant.register_sidechain_asset) ) will be burned. A SORA mainnet asset (registered via [register_thischain_asset](https://sora-xor.github.io/sora2-network/master/doc/substrate_gen/runtime/runtime_types/parachain_bridge_app/pallet/enum.Call.html#variant.register_thischain_asset)) will be reserved._
+
+### Parachain to SORA
+
+```mermaid
+sequenceDiagram
+    actor A as Alice
+    participant P as Parachain
+    participant R as Relayer
+    participant S as SORA
+
+    A ->> P: Send tokens via XCM¹
+    P ->> P: Add message to commitment
+    P ->> R: Prepare commitment
+    R ->> R: Sign commitment
+    R ->> P: Approve commitment
+    P ->> R: Get signatures
+    R ->> S: Submit commitment
+    S ->> A: Mint/Send tokens
+```
+
+_¹ The assets can originate from various sources, including other parachains or the Relay Chain, and the resulting action (reserve or burn) on the source chain is dependent on its specific implementation (typically initiated by calling either `xcmPallet.reserveTransferAssets` or `xTokens.transfer`). Ultimately, SORA parachain remains unaware of the final outcome, as it only receives an XCM message that will be subsequently relayed to the mainnet._
+
+## Common Pallets
+
+There are several common pallets used in the SORA project, providing bridge functionality on both the parachain and SORA mainnet sides:
+
+- **SubstrateDispatch**: Developed by SORAMITSU ([GitHub link](https://github.com/sora-xor/sora2-common/tree/develop/pallets/dispatch)). This pallet decodes and checks messages, performs runtime calls, and creates a bridge origin to execute **XCMApp** and **ParachainBridgeApp** extrinsics. Only the bridge origin can call these extrinsics.
+
+- **SubstrateBridgeInboundChannel**: Developed by SORAMITSU ([GitHub link](https://github.com/sora-xor/sora2-common/tree/develop/pallets/substrate-channel/src/inbound)). This pallet handles the submission of incoming messages. While submitting messages can be called from any account, a relayer is used for this function. It utilizes **BeefyLightClient** or **MultisigVerifier** for message verification and **SubstrateDispatch** for message calling.
+
+- **SubstrateBridgeOutboundChannel**: Developed by SORAMITSU ([GitHub link](https://github.com/sora-xor/sora2-common/tree/develop/pallets/substrate-channel/src/outbound)). This pallet is responsible for sending outgoing messages. It is used by the **ParachainBridgeApp** and **XCMApp** to send messages to other networks.
```

Comment:
I think you can just say `source code`. You just need something more specific/descriptive than just `GitHub link`.

please also note that if you apply this suggestion, there are several similar instances below where you need to apply the same change as here (for other pallets)

```suggestion
- **SubstrateDispatch**: Developed by SORAMITSU ([source code](https://github.com/sora-xor/sora2-common/tree/develop/pallets/dispatch)). This pallet decodes and checks messages, performs runtime calls, and creates a bridge origin to execute **XCMApp** and **ParachainBridgeApp** extrinsics. Only the bridge origin can call these extrinsics.

- **SubstrateBridgeInboundChannel**: Developed by SORAMITSU ([source code](https://github.com/sora-xor/sora2-common/tree/develop/pallets/substrate-channel/src/inbound)). This pallet handles the submission of incoming messages. While submitting messages can be called from any account, a relayer is used for this function. It utilizes **BeefyLightClient** or **MultisigVerifier** for message verification and **SubstrateDispatch** for message calling.

- **SubstrateBridgeOutboundChannel**: Developed by SORAMITSU ([source code](https://github.com/sora-xor/sora2-common/tree/develop/pallets/substrate-channel/src/outbound)). This pallet is responsible for sending outgoing messages. It is used by the **ParachainBridgeApp** and **XCMApp** to send messages to other networks.
```

Reply:
Added as part of https://github.com/sora-xor/sora-docs/pull/44/commits/984d2210d8c1c290205c8b1c357a61816a71155a